lib/utils/parse/parse_test.go

```diff
@@ -211,9 +211,16 @@ func TestVariable(t *testing.T) {
 // TestInterpolate tests variable interpolation
 func TestInterpolate(t *testing.T) {
 	t.Parallel()
+
+	errCheckIsNotFound := func(tt require.TestingT, err error, i ...interface{}) {
```

Comment:
Looks like `tt` is not used?

Reply:
The same with `i`

Reply:
Oops :man_facepalming: 
`tt` should be used instead of `t`

`i` is not necessary because we don't care about extra arguments
It's only meant to satisfy the https://pkg.go.dev/github.com/stretchr/testify/require#ErrorAssertionFunc interface